website/src/blogs/2017-06-28-storage-is-dead-long-live-openebs.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Storage is Dead! Long live OpenEBS
+author: Evan Powell
+author_info: Founding CEO of a few companies including StackStorm (BRCD) and Nexenta — and CEO & Chairman of OpenEBS/MayaData. ML and DevOps and Python, oh my!
+tags: DevOps, Kubernetes
+date: 28-06-2017
+excerpt: In this blog I share something I’ve been saying for a while now that people suggested might just be audacious enough to spark a conversation.
+---
+
+In this blog, I share something I’ve been saying for a while now that people suggested might just be audacious enough to spark a conversation.
+
+Plus — I explain why I’ve decided to help found another storage project — [OpenEBS](http://www.openebs.io/) — and to become CEO of the underlying company, [CloudByte](http://www.cloudbyte.com/), at a time in which venture capital funding to space has declined (at least if you accept Rubrik and [Cloudian](https://cloudian.com/) and similar) funding as outliers :)).
+
+TL;DR: the old storage industry is ****dying**** like many stagnant and declining IT sectors before it, killed through outdated business practices and architectures and the emergence of a new set of personas that have radically different requirements, expectations, and even tastes. The new architectures and business models already exist — though they are unevenly distributed.
```

Comment:
please check once if TL;DR should be Tl;dr ??

Reply:
It is TL;DR https://en.wikipedia.org/wiki/Wikipedia:Too_long;_didn%27t_read